### PR TITLE
[NITF] [FORMATTER] use local time for filename generation

### PR DESCRIPTION
**Pull request by jerome-poisson**
improvment on SDNTB-264

### CODE REVIEW

File: server/ntb/publish/ntb_nitf.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def _get_filename(self, article):
         example: 2016-08-16_11-07-46_Nyhetstjenesten_Innenriks_ny1-rygge-nedgang.xml
         """
         metadata = {}
-        metadata['date'] = article['versioncreated'].strftime("%Y-%m-%d_%H-%M-%S")
+        metadata['date'] = article['versioncreated'].astimezone().strftime("%Y-%m-%d_%H-%M-%S")
```
Comment (petrjasek) [inline server/ntb/publish/ntb_nitf.py]: is it using server timezone?

Comment (jerome-poisson) [inline server/ntb/publish/ntb_nitf.py]: It's using server's local time zone yes. Actually it was not specified and UTC was used here, but as local timezone is used in other NTB metadata, I've changed it too for consistency.

**Comment by ioanpocol** [comment]
:+1:

File: server/ntb/publish/ntb_nitf.py (hunk 1 above)
Comment (petrjasek) [inline server/ntb/publish/ntb_nitf.py]: imo it would be better then to use zone from config - there is `DEFAULT_TIMEZONE` setting

Comment (jerome-poisson) [inline server/ntb/publish/ntb_nitf.py]: oh I didn't knew there was this in settings, yes it makes definitely more sense then, I'll update the P.R.

**Comment by jerome-poisson** [comment]
following P.R. will replace this one: https://github.com/superdesk/superdesk/pull/2282, so I'm closing it.

**Pull request closed without merging**